pycobertura/reporters.py

```diff
@@ -1,90 +1,64 @@
-from collections import namedtuple
 from jinja2 import Environment, PackageLoader
 from pycobertura.cobertura import CoberturaDiff
-from pycobertura.utils import green, rangify, red
+from pycobertura.utils import green, red, stringify
 from pycobertura.templates import filters
 from tabulate import tabulate
 
 
 env = Environment(loader=PackageLoader("pycobertura", "templates"))
 env.filters["line_status"] = filters.line_status
 env.filters["line_reason"] = filters.line_reason_icon
+env.filters["is_not_equal_to_dash"] = filters.is_not_equal_to_dash
+env.filters["misses_color"] = filters.misses_color
 
-row_attributes = ["filename", "total_statements", "total_misses", "line_rate"]
-file_row = namedtuple("FileRow", row_attributes)
-file_row_missed = namedtuple("FileRowMissed", row_attributes + ["missed_lines"])
+headers_without_missing = ["Filename", "Stmts", "Miss", "Cover"]
+headers_with_missing = ["Filename", "Stmts", "Miss", "Cover", "Missing"]
 
 
 class Reporter(object):
```

Comment:
Since we only support py3 now, should we remove `object`?

```suggestion
class Reporter:
```

What do you think? If yes then we should do this in a separate PR as a follow-up to #123.

Reply:
I think that would be good :)

Reply:
made a PR here: https://github.com/aconrad/pycobertura/pull/126. Also includes PRs from #123 (only the last 2 commits should matter)

Reply:
Ok that PR #126 looks more review-friendly now:)

Reply:
Oh wow, awesome, thanks!